src/metabase/models/cloud_migration.clj

```diff
@@ -0,0 +1,213 @@
+(ns metabase.models.cloud-migration
+  "A model representing a migration to cloud."
+  (:require
+   [cheshire.core :as json]
+   [clj-http.client :as http]
+   [clojure.java.io :as io]
+   [clojure.set :as set]
+   [metabase.cmd.dump-to-h2 :as dump-to-h2]
+   [metabase.models.interface :as mi]
+   [metabase.models.setting :refer [defsetting]]
+   [metabase.models.setting.cache :as setting.cache]
+   [metabase.util.i18n :refer [deferred-tru tru]]
+   [metabase.util.log :as log]
+   [methodical.core :as methodical]
+   [toucan2.core :as t2]
+   [toucan2.pipeline :as t2.pipeline])
+  (:import
+   [java.io File InputStream]))
+
+(set! *warn-on-reflection* true)
+
+(def CloudMigration "Cloud Migration" :model/CloudMigration)
```

Comment:
we shouldn't define this for new models, we should just use `:model/CloudMigration` anywhere.

we have this for old models because when we migrate to toucan2 we didnt' want to update every single reference we have in the code base.

Reply:
Removed